integration/setup/setup.go

```diff
@@ -69,5 +72,19 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	// Ensure Chart CRD creation
+	{
+		c := k8scrdclient.Config{
+			K8sExtClient: config.Host.ExtClient(),
+			Logger:       config.Logger,
+		}
+
+		var crdClient *k8scrdclient.CRDClient
+		crdClient, err = k8scrdclient.New(c)
+		crdBackoff := backoff.NewMaxRetries(3, 1*time.Second)
```

Comment:
Backoff is very short here. If it works then fine I just don't want the test to be flakey if sometimes it takes longer.

Reply:
I run off test almost 30 times appx. but didn't get any situations in here.

Reply:
OK cool that's fine then.